File: CodeForces/1701-1800/1742/A/1742A.py
# Problem 1742A: Plus or Minus
# https://codeforces.com/contest/1742/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: October 22nd, 2025
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

w: int = int(input())

for i in range(0, w):
    numbers: list[int] = list(map(int, input().split()))
    numbers.sort()
    if numbers[0] + numbers[1] == numbers[2]:
        print("YES")
    else:
        print("NO")

File: CodeForces/1701-1800/1742/A/1742A.cpp
// Problem 1742A: Plus or Minus
// https://codeforces.com/contest/1742/problem/A
// Compiled with: g++ (Ubuntu 15.2.0-4ubuntu4) 15.2.0
// Linux version: Linux kernel 6.17.0-5-generic
// Submitted on: October 25th, 2025
// Codeforces language used: GNU G++23 14.2 (64 bit, msys2)

#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main() {
    int n;
    cin >> n;

    for (int i = 0; i < n; i++) {
        vector<int> arr(3);
        for(int i = 0; i < 3; i++) {
            cin >> arr[i];
        }
        sort(arr.begin(), arr.end()); 
        if (arr[0] + arr[1] == arr[2]) {
            cout << "YES\n";
        } else {
            cout << "NO\n";
        }
    }
    
}
